rs/cli/src/registry_dump.rs

```diff
@@ -54,8 +57,20 @@ pub async fn dump_registry(path: &Option<PathBuf>, network: &Network, version: &
         Err(e) => return Err(e.into()),
     };
 
-    let nodes = get_nodes(&local_registry, version, &node_operators, &subnets)?;
-
+    let nodes = get_nodes(&local_registry, version, &node_operators, &subnets, network).await?;
+    // Calculate number of rewardable nodes for node operators
+    for node_operator in node_operators.values_mut() {
+        node_operator.total_up_nodes = nodes
+            .iter()
+            .filter(|n| {
+                n.node_operator_id == node_operator.node_operator_principal_id && (n.status == Status::Healthy || n.status == Status::Degraded)
```

Comment:
WDYT about checking `n.status != Status::Dead`. This should in the worst case show that all nodes are alive (if we can't get the node health). The way it's now, we'd mark all nodes as dead if we can't get node health.

Reply:
Code for mapping from public dashboard:
```rust
impl Status {
    pub fn from_str_from_dashboard(s: &str) -> Self {
        match s {
            "UP" | "UNASSIGNED" => Self::Healthy,
            "DEGRADED" => Self::Degraded,
            "DOWN" => Self::Dead,
            _ => Self::Unknown,
        }
    }
}
```
The only thing that i am afraid of is the `_ => Self::Unknown` meaning that if dashboard adds something that we don't map it will give false calculations. I'd rather add one more `||` than to add a `n.status != Status::Dead`

Reply:
Alright, let's do that, and make the tool break early if something changes upstream.